Sources/Commands/Options.swift

```diff
@@ -46,33 +46,43 @@ struct GlobalOptions: ParsableArguments {
 struct LocationOptions: ParsableArguments {
     init() {}
 
-    /// The custom build directory, if provided.
-    @Option(help: "Specify build/cache directory")
-    var buildPath: AbsolutePath?
+    /// The custom .build directory, if provided.
+    @Option(name: .customLong("scratch-space-path"), help: "Specify the scratch space directory (default .build)", completion: .directory)
+    var _scratchSpacePath: AbsolutePath?
 
-    @Option(help: "Specify the shared cache directory")
-    var cachePath: AbsolutePath?
+    @Option(name: .customLong("build-path"), help: .hidden)
+    var _deprecated_buildPath: AbsolutePath?
 
-    @Option(help: "Specify the shared configuration directory")
-    var configPath: AbsolutePath?
+    var scratchSpaceDirectory: AbsolutePath? {
+        self._scratchSpacePath ?? self._deprecated_buildPath
+    }
+
+    @Option(name: .customLong("cache-path"), help: "Specify the shared cache directory", completion: .directory)
+    var cacheDirectory: AbsolutePath?
 
-    @Option(help: "Specify the shared security directory")
-    var securityPath: AbsolutePath?
+    @Option(name: .customLong("config-path"), help: "Specify the shared configuration directory", completion: .directory)
+    var configurationDirectory: AbsolutePath?
 
-    /// The custom working directory that the tool should operate in (deprecated).
-    @Option(name: [.long, .customShort("C")])
-    var chdir: AbsolutePath?
+    @Option(name: .customLong("security-path"), help: "Specify the shared security directory", completion: .directory)
+    var securityDirectory: AbsolutePath?
 
     /// The custom working directory that the tool should operate in.
-    @Option(help: "Change working directory before any other operation")
-    var packagePath: AbsolutePath?
+    @Option(name: .customLong("package-path"), help: "Change working directory before any other operation", completion: .directory)
```

Comment:
It's actually a bit surprising that this is implemented as changing directory instead of just referencing a package.  I think I was able to mitigate that for plugins by having them start in the original working directory, e.g. for a plugin invoked from the command line it's natural for relative arguments to be relative to the current working directory.  But I thought that was just an implementation shortcut and not how it's officially documented.

Reply:
I see now that `git -C` works this way, and perhaps it makes sense.  Interestingly, `swift run` doesn't seem to work that way, as `swift run --package-path /tmp/MyExec MyExec` will run in the original working directory (which is what I'd expect).

Reply:
this PR does not change this functionality - it only cleans up the way the CLI flags are organized. material for future PR?

Reply:
Yes, good point, this is unrelated to this PR.  I don't know whether that qualifies as a bug to be fixed or not — it just seemed surprising to me when I discovered it.